ftpbench/ftpbench.py

```diff
@@ -227,7 +231,11 @@ def _check():
     gr_pool = Pool(size=opts["concurrent"])
     try:
         with Timeout(opts["maxrun"] * 60 or None):
-            while True:
+            if opts["maxiter"] == -1:
+                iterator = count()
+            else:
+                iterator = range(opts["maxiter"])
```

Comment:
Please get this logic out of the function scope. It might be put in the config parsing section to avoid being repeated multiple times.